DIP/Example5.md

```diff
@@ -0,0 +1,22 @@
+### Usage
+
+---
+#### Where
+https://github.com/angular/angular/blob/main/packages/router/src/router.ts#L568 
```

Comment:
Yes, but I meant only Ingector. It's kind of helper of getting dependencies.

Reply:
I agree we can say that Angular supports dependency injection technique which helps to comply with DIP. But maybe `Router` just not the best example, because it is a high-level module that depends on a low-level one.

Reply:
👍